lib/active_admin_sidebar/positions.rb

```diff
@@ -1,10 +1,28 @@
 module ActiveAdminSidebar
   module Positions
-    def left_sidebar!
-        @sidebar_position = :left
+    def left_sidebar!(options = {})
+      @sidebar_options = {position: :left}
+      if options.fetch(:collapsed, false)
+        collapsed_sidebar
+        @sidebar_options.merge!(
+            is_collapsed: session[:collapsed_sidebar],
+            collapsed: true
+        )
+      end
     end
+
     def right_sidebar!
-        @sidebar_position = :right
+      @sidebar_options = {position: :right}
+    end
+
+    def collapsed_sidebar
+      if request.xhr?
+        if params[:collapsed_sidebar].present?
+          collapsed = ActiveRecord::ConnectionAdapters::Column::TRUE_VALUES.include? params[:collapsed_sidebar]
+          session[:collapsed_sidebar] = collapsed
+          render json: {collapsed_sidebar: collapsed} and return
```

Comment:
Use `&&` instead of `and`.<br>Space inside { missing.<br>Space inside } missing.